### PR TITLE
Adds express charset advisory

### PR DESCRIPTION
**Pull request by tomsteele**
How does nsp handle stuff like "< 3.8.0, < 4.3.0" @evilpacket

### CODE REVIEW

**Comment by dougwilson** [comment]
Can this be published? Version ranges work just fine in the API.

**Comment by evilpacket** [comment]
Yup. I had it in my brain that I already did this. I'll get to it tomorrow!

**Comment by evilpacket** [comment]
@dougwilson I don't see that this actually works as we need it to. I merged it but haven't pushed it live yet as I don't think the versions as specified in this pr work properly. Thoughts?

**Comment by dougwilson** [comment]
Oh, you need to replace the commas with ||

File: advisories/express-no-charset-in-content-type-header.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: express No Charset in Content-Type Header
+author: Paweł Hałdrzyński
+module_name: express
+publish_date: Fri Sep 12 2014 07:46:45 GMT-0700 (PDT)
+cves: "[{\"name\":\"CVE-2014-6393\",\"link\":\"https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2014-6393\"}]"
+vulnerable_versions: "< 3.11.0, < 4.5.0"
```
Comment (dougwilson) [inline advisories/express-no-charset-in-content-type-header.md]: Change this to

    <3.11 || >= 4 <4.5

File: advisories/express-no-charset-in-content-type-header.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+patched_versions: ">= 3.11.0, >= 4.5.0"
```
Comment (dougwilson) [inline advisories/express-no-charset-in-content-type-header.md]: Change this to

    >=3.11 <4 || >=4.5

**Comment by evilpacket** [comment]
Gotcha. I tried to replace with || but ended but because of the <4.5 it would trip on 3.11 3.12 etc. Let me test.

**Comment by dougwilson** [comment]
Ok, I left line notes. If the ones I put there still don't work, I'll prob them when I get home in a couple hours :) The serve-static advisory is similar, so you may be able to go off that as well.

**Comment by evilpacket** [comment]
@dougwilson thanks for the help. I semver bad.

**Comment by dougwilson** [comment]
lol :) I'm happy to help, because I love this service.

**Pull request merged**